virtual_rainforest/models/abiotic/wind.py

```diff
@@ -0,0 +1,551 @@
+"""The `abiotic.wind` module.
```

Comment:
With the new documentation setup you shouldn't put a heading for module docstrings as `sphinx` adds a heading itself. The `# noqa: D205, D415` flags at the bottom of the docstring stop `flake8` from complaining about this